apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -135,6 +135,9 @@ defmodule AndiWeb.EditLiveView do
 
     AndiWeb.Endpoint.broadcast_from(self(), "form-save", "save-all", %{dataset_id: dataset_id})
 
+    # Todo: Rearchitect how concurrent form events are handled and remove these sleeps from draft-save and publish of datasets and ingestions
+    Process.sleep(1_000)
```

Comment:
Is this needed for your card or are these changes from my branch? I removed some of these sleeps before merging my branch.